wordpress-mobile/WordPress-iOS
# Dark Mode: initial commit

Refs #12320 

This is the intial dark mode commit, which lays the foundations of the rest of the changes. This initial set was focussed on creating these major functionality screenshots:

Stats | Reader | Notifications
--- | --- | ---
![Simulator Screen Shot - iPhone Xʀ - 2019-08-14 at 19 24 58](https://user-images.githubusercontent.com/517257/63061303-86184a00-bea9-11e9-9e01-2406011541c7.png) | ![Simulator Screen Shot - iPhone Xʀ - 2019-08-12 at 15 35 30](https://user-images.githubusercontent.com/517257/63061372-c11a7d80-bea9-11e9-9baf-a9de071f8fae.png) | ![Simulator Screen Shot - iPhone Xʀ - 2019-08-14 at 12 36 15](https://user-images.githubusercontent.com/517257/63061316-992b1a00-bea9-11e9-9bd4-e13e8f9572e9.png)

To test:
- Build and run in Xcode 11
- Switch to dark mode (Settings > Developer in the Simulator)
- Check the screens posted above
- Ensure they basically match the mockups. You'll see lots of white backgrounds, but these screens should be more-or-less accurate, with perhaps some remaining issues when scrolling down

Update release notes:
